server/controllers/gainzController.js

```diff
@@ -0,0 +1,82 @@
+const db = require('../server/postgresPool.js');
+
+const gainzController = {};
+
+gainzController.getExcercises = (req, res, next) => {
+  const exerciseQ = 'SELECT DATA FROM EXERCISE TABLE';
+  db.query(exerciseQ)
+    .then(data => {
+      res.locals.excerciseQuery = data.rows;
+      next();
+    })
+    .catch(err => {
+      console.log(err);
+      next();
+    });
+};
+
+
+gainzController.getHistory = (req, res, next) => {
+  const drillQuery = 'SELECT DATA FROM DRILL TABLE';
+  db.query(drillQuery)
+    .then(data => {
+      res.locals.drillQuery = data.rows[0];
```

Comment:
Why only grabbing one data item? Don't we need an array of all items for history?